data/data-pipeline/data_pipeline/etl/sources/census_acs_median_income/etl.py

```diff
@@ -112,7 +112,7 @@ def _transform_geocorr(self) -> pd.DataFrame:
 
         # Drop all the duplicated rows except for the first one (which will have the place name):
```

Comment:
This actually needs to be changed -- right now it'll give us the wrong data because the source data is from block groups.